lib/testing/src/lib/core/pages/data-table/data-table-builder.ts

```diff
@@ -15,15 +15,14 @@
  * limitations under the License.
  */
 
-import { Column } from './column';
 import { TextColumn } from './text-column';
 import { DateColumn } from './date-column';
 import { DataTableItem } from './data-table-item';
 import { ElementFinder } from 'protractor';
 
 export class DataTableBuilder {
 
-    createDataTable(columns: Array<Column>, rootElement?: ElementFinder): DataTableItem {
+    createDataTable(columns: Array<any>, rootElement?: ElementFinder): DataTableItem {
```

Comment:
why is it any now?

Reply:
I changed with any because i am giving an array like this:     defaultColumns = [
        { columnName: this.columnNames.size, columnType: 'text' }, .... ]. And i get an error saying that "Type '{ columnName: string; columnType: string; }' is missing the following properties from type 'Column': locator, createLocator". I fixed this by using any. Should i have done differently?